index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  </head>
+
+  <body>    
+        <main id="main">
+            <div id="title">Dr. Norman Borlaug</div>
+            <div id="img-div">         
+                <img src="./images/Dr_Norman.jpeg" alt="Dr Norman">
+            </div>
```

Comment:
Kindly let all your comments include an explanation, a "why", for each requested change. This will help them see the reasons with you. Some reasons might seem obvious to you but not to the author of the code.